integration-tests/smoke/vrfv2_test.go

```diff
@@ -25,7 +26,7 @@ import (
 	uuid "github.com/satori/go.uuid"
 )
 
-var _ = Describe("VRFv2 suite @v2vrf", func() {
+var _ = Describe("VRFv2 suite @vrfv2", func() {
```

Comment:
This means that the `@vrf` tag will grab this too. So vrfv2 test will get run twice.